src/backend/base/langflow/components/__init__.py

```
@@ -0,0 +1,14 @@
+from langflow.components.brightdata import (
+    BrightDataSearchEngineComponent,
+    BrightDataStructuredDataEnhancedComponent,
+    BrightDataWebScraperComponent,
+)
+
+# Add to the __all__ list (in alphabetical order):
+__all__ = [
+    # ... existing components ...
+    "BrightDataSearchEngineComponent",
+    "BrightDataStructuredDataEnhancedComponent", 
```

Ruff (0.11.9)

Trailing whitespace

Remove trailing whitespace

(W291)